src/main/scala/net/ceedubs/ficus/readers/ArbitraryTypeReader.scala

```diff
@@ -22,12 +59,13 @@ class ArbitraryTypeReaderMacros(val c: blackbox.Context) extends ReflectionUtils
       new ValueReader[T] {
         def read(config: Config, path: String): T = instantiateFromConfig[T](
           config = c.Expr[Config](Ident(TermName("config"))),
-          path = c.Expr[String](Ident(TermName("path")))).splice
+          path = c.Expr[String](Ident(TermName("path"))),
+          mapper = c.Expr[NameMapper](q"""net.ceedubs.ficus.readers.NameMapper()""")).splice
```

Comment:
I think you want to add `_root_` here to preserve hygiene (since someone could use this inside a package which contains a subpackage named `net`. See the notes on hygiene here http://docs.scala-lang.org/overviews/quasiquotes/hygiene.